crt_portal/static/sass/custom/form.scss

```diff
@@ -270,6 +271,7 @@ form#report-form {
         margin-top: 0;
         margin-bottom: units($spacer-1x);
         padding-left: units($spacer-5x);
+        border-bottom: 1px solid #dcdee0;
```

Comment:
I believe Jacklynn asked for #DFE1E2 -- also, please use a SCSS variable. You can look up the colors in the [USWDS system color page](https://designsystem.digital.gov/design-tokens/color/system-tokens/). This color turns out to be defined as `gray-cool-10` and we already have this defined as $theme-color-base-lighter in `_uswds-theme-color.scss`, so here it would be `color($theme-color-base-lighter)`